guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -13,6 +13,7 @@ ifdef::satellite[]
 endif::[]
 ifdef::foreman-el,katello,orcharhino,satellite[]
 * xref:migrating-ansible-content_{context}[]
+* xref:post_upgrade-updating-templates-parameters_{context}[]
```

Comment:
Move this one line down, below the `endif`, because the referenced module is included for all products.